builder/llpcPipeline.h

```diff
@@ -49,6 +49,102 @@ using namespace llvm;
 
 class BuilderContext;
 
+// =====================================================================================================================
+// Structs for setting pipeline state
+
+// Bit values of NGG flags. This is done as bit values rather than bitfields so the flags word appears
+// in a platform-independent way in IR metdata.
+enum NggFlag : uint32_t
```

Comment:
can you reuse the enum and struct in llpc.h?

Reply:
I could do that, but I decided not to, because there is not really any justification for having NGG options in a separate struct from other pipeline options, and there are so many NGG options that they work better as (mostly) bits in a single value like this.